### PR TITLE
Add MPI-EVA Cluster profile

### PR DESCRIPTION
**Pull request by jfy133**
Currently only for DAG.

Adds eager specific pipeline too.

Sorry for commit mess. Java sucks.

### CODE REVIEW

File: nfcore_custom.config [hunk 1]
```diff
@@ -23,7 +23,8 @@ profiles {
   crick        { includeConfig "${params.custom_config_base}/conf/crick.config" }
   czbiohub_aws { includeConfig "${params.custom_config_base}/conf/czbiohub_aws.config" }
   ebc          { includeConfig "${params.custom_config_base}/conf/ebc.config" }
-  eddie	 { includeConfig "${params.custom_config_base}/conf/eddie.config" }
+  eddie	       { includeConfig "${params.custom_config_base}/conf/eddie.config" }
```
Comment (jfy133) [inline nfcore_custom.config:26]:
```suggestion
  eddie       { includeConfig "${params.custom_config_base}/conf/eddie.config" }
```

**Comment by maxibor** [comment]
Be sure to squash when merging though ;) https://cloudfour.com/thinks/squashing-your-pull-requests/

File: nfcore_custom.config [hunk 2]
```diff
@@ -23,7 +23,8 @@ profiles {
   crick        { includeConfig "${params.custom_config_base}/conf/crick.config" }
   czbiohub_aws { includeConfig "${params.custom_config_base}/conf/czbiohub_aws.config" }
   ebc          { includeConfig "${params.custom_config_base}/conf/ebc.config" }
-  eddie	 { includeConfig "${params.custom_config_base}/conf/eddie.config" }
+  eddie       { includeConfig "${params.custom_config_base}/conf/eddie.config" }
```
Comment (jfy133) [inline nfcore_custom.config:26]:
```suggestion
  eddie        { includeConfig "${params.custom_config_base}/conf/eddie.config" }
```

**Review by maxibor — APPROVED** [review]

**Comment by jfy133** [comment]
Closed in lieu of #224

**Pull request closed without merging**